ci/clang_check_attributes.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 by Antmicro. Verilator is free software; you
```

Comment:
I'd prefer it be copyright under my name as the rest of the package, but if you all feel strongly this can work - but then change "Verilator is" to "This is"

Reply:
Sure, I've changed copyright to your name.